.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: Test
+name: CPU CI
```

Comment:
Make sure to update the docs-preview-pr automation to depend on this new name: https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/92baa7d8a2eeed9d937d01624e159eaaaa371024/.github/workflows/docs-preview-pr.yaml#L5